### ISSUE TITLE
Q1: Different solution

### ISSUE THREAD

**Issue by MiloshBo** [zhiwehu/Python-programming-exercises#29]
Mine solution is:

```
for i in range(2000, 3201):
    if i%7 == 0 and i%5 != 0:
        print(i, end = ', ')
```
Is there any particular reason why given solution with .append and .join is used, is it better then mine from some reason? Because the one I wrote takes less lines. I'm a complete beginner, apologize if my question is ridiculous.

**Comment by johnashu** [comment]
No.  I think yours is better considering the questions requirements.  The solution is correct either way.

**Comment by Ajaychhettri** [comment]
this is good

**Comment by NegarAmirsalari** [comment]
Good

**Comment by Abhijeet199** [comment]
Nice what about this guys
lst = [str(i) for i in range(2000, 3201) if i % 7 == 0 and i % 5 != 0]
print(', '.join(lst))
